packages/framework-integration-tests/integration/provider-specific/local/start/start.integration.ts

```diff
@@ -0,0 +1,16 @@
+//import { AWSTestHelper } from '@boostercloud/framework-provider-aws-infrastructure'
+//import { applicationName } from '../../../helper/app-helper'
+import { expect } from '../../../helper/expect'
+
+describe('After start', () => {
+  describe('the local provider', () => {
+    it('has been started successfully', async () => {
+      // The project must have been deployed by the cliHelper hook in setup.ts
+      // that scripts uses the cli to do the deployment, so we just check here
+      // that the Cloudformation was run by AWS successfully. For that, we can just
+      // build the AWSHelper. It will throw if the AWS stack is not ready.
+      //await expect(AWSTestHelper.build(applicationName())).to.be.eventually.fulfilled
+      expect(true).to.be.true
```

Comment:
this is temporal. We need to ensure the Express server to be up and running. Maybe we can get the GraphQL schema with Apollo when we run `LocalTestHelper.build(applicationName())` or create a /health endpoint in the provider infrastructure

Reply:
Maybe you can do a `ping` to the server, although I think that what you say about doing a harmless graphQL query (for the schema for example) is a good solution.

In any case, I don't know if you are still working on this PR but, if not, Could you please remove the AWS-specific comment and, if you are not going to tackle this test now, create an issue and put a link in a comment here?